src/datachain/dataset.py

```diff
@@ -260,9 +261,17 @@ def serialized_schema(self) -> dict[str, Any]:
             for c_name, c_type in self.schema.items()
         }
 
+    @cached_property
+    def preview(self) -> Optional[list[dict]]:
+        if isinstance(self._preview_data, str):
+            return json.loads(self._preview_data)
```

Comment:
Should we use `orjson` as it is faster?
See https://github.com/iterative/datachain/pull/178